am-ui/src/app/components/search-result-details/search-result-details.component.ts

```diff
@@ -10,25 +12,55 @@ import {Subscription} from 'rxjs';
 export class SearchResultDetailsComponent implements OnInit, OnDestroy {
   public language: String = '';
   public subscription: Subscription;
+  public searchResultDetailSubscription: Subscription;
   public result = {};
+  public resultJson: any = [];
+  public id: string | null;
 
   /**
    * @description Component to display the detail view of a Topic
    * @constructor
    * @param {TranslateService} translate - use of translate service to detect language change
    */
-  constructor(private translate: TranslateService) {}
+  constructor(
+    private translate: TranslateService,
+    private route: ActivatedRoute,
+    private searchDetailResultService: SearchResultDetailsService
+  ) {
+    this.getDefaultSearchResultDetails();
+  }
+
+  getDefaultSearchResultDetails() {
+    this.id = this.route.snapshot.paramMap.get('id');
+    if (this.id !== null) {
+      /*eslint-disable-next-line*/
```

Comment:
Remove the eslint-disable.
Line 38 should be:`.getById(+this.id).subscribe(res => { `